request.go

```diff
@@ -333,6 +348,12 @@ func (r *Request) SetResult(res interface{}) *Request {
 	return r
 }
 
+// This method is to register curl cmd for request executed.
+func (r *Request) SetResultCurlCmd(curlCmd *string) *Request {
```

Comment:
@ahuigo - please remove this method and the related implementation code. 

Resty could provide a cURL command generation on-demand instead of implicit via -
```go
resp, err := client.R().
     EnableTrace().
     Get("https://httpbin.org/get")

curlCmd := resp.Request.GenerateCurlCommand()
```

Reply:
done